src/views/simplified-account/settings/card-payments/apple-pay.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Card payments
+{% endblock %}
```

Comment:
This needs to be updated to the new title format with `{% set settingsPageTitle = '...' %}`, page titles should be listed on the designs - similarly on other settings pages

Reply:
Updated to ues new style.